app/src/main/AndroidManifest.xml

```diff
@@ -110,6 +110,17 @@
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="com.intervaltimer.google.activities.MainActivity" />
+
```

Comment:
How do you decide where to add the intent? E.g., why is starting added to the main and stopping added to this one?

Reply:
The intent is added to the workout that handles it. If we start the activity, we need to use the information stored in MainActivity to launch the WorkoutActivity; however, if we stop the activity, we normally want the existing and active WorkoutActivity to show the finishing screen. This is just the logical way to distribute the responsibilities IMO.

Reply:
I fully agree!

What about a very short comment in the corresponding xml code to explain this, e.g.,

"Handle start commands in the MainActivity to launch the WorkoutAcitivity from there."

?